t/unix.t

```diff
@@ -0,0 +1,142 @@
+use strict;
+use warnings;
+use Test::More tests=>45;
```

Comment:
For this dist I've standardized on `Test::V0` so `Test::More` should not be used.

Reply:
I think you have a typo there..  Did you mean Test2::V0?

Reply:
yes.